scripts/check.sh

```diff
@@ -16,7 +16,7 @@ fi
 
 stack install stylish-haskell
 changed=()
-for file in $(git diff --name-only $UPSTREAM_BRANCH)
+for file in $(git diff --name-only --diff-filter=d $UPSTREAM_BRANCH)
```

Comment:
why is this?

Reply:
This is from a previous PR I already merged to master: https://github.com/kframework/kore/pull/972